src/_includes/partials/pageList.njk

```diff
@@ -81,6 +81,7 @@
 			</p>
 			{% endif %}
 			<p><strong>{{ pageList[locale].descriptionText }}{% if locale === "fr" %} {% endif %}:</strong> {{ entry.data.description | striptags(true) | escape | nl2br }}</p>
+			<p><strong>{{ pageList[locale].descriptionNoneText }}{% if locale === "fr" %} {% endif %}:</strong> {{ entry.data.description | striptags(true) | escape | nl2br }}</p>
```

Comment:
I was looking for an `if` condition that would show `{{ landingPage[locale].descriptionNoneText }}` if there's nothing in the `description:` data

```
<p><strong>{{ pageList[locale].descriptionText }}
{% if locale === "fr" %} {% endif %}:</strong>
{% if description %}
{{ entry.data.description | striptags(true) | escape | nl2br }}
{% else %}
{{ landingPage[locale].descriptionNoneText }}
{% endif %}</p>
```